src/System.Private.CoreLib/shared/System/Diagnostics/StackTrace.cs

```diff
@@ -317,14 +203,8 @@ internal enum TraceFormat
         internal string ToString(TraceFormat traceFormat)
```

Comment:
Do you expect that you will be able to use this implementation of ToString for Mono AOT?

The CoreRT ToString implementation is quite different because of AOT, and I do not think this one will work well for CoreRT.